samples/bluetooth/peripheral_hr/prj_minimal.conf

```diff
@@ -80,7 +80,7 @@ CONFIG_ISR_STACK_SIZE=1024
 
 # Disable features not needed
 CONFIG_TIMESLICING=n
-CONFIG_MINIMAL_LIBC_MALLOC=n
+CONFIG_COMMON_LIBC_MALLOC=n
```

Comment:
I just realised that we can no longer disable the `malloc` implementation when using the minimal libc after the changes in this PR because `CONFIG_COMMON_LIBC_MALLOC` does not have a prompt (Kconfigs without a prompt is not user configurable and cannot be set here).

Reply:
odd that we didn't get any errors while testing. Thanks for figuring this out.